src/rgw/rgw_lua_utils.h

```diff
@@ -0,0 +1,168 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <ctime>
+#include <lua.hpp>
+
+class CephContext;
+
+namespace rgw::lua {
+
+// push ceph time in string format: "%Y-%m-%d %H:%M:%S"
+template <typename CephTime>
+void lua_pushtime(lua_State* L, const CephTime& tp) 
+{
+  const auto tt = CephTime::clock::to_time_t(tp);
+  const auto tm = *std::localtime(&tt);
+  char buff[64];
+  std::strftime(buff, sizeof(buff), "%Y-%m-%d %H:%M:%S", &tm);
```

Comment:
consider using `ceph::to_iso_8601()` here

Reply:
my function is templatized to support both `ceph::real_clock` and `ceph::coarse_real_clock`.
we can probably do the same for `ceph::to_iso_8601()` but would like to do that in a separate PR